RELEASE_NOTES.md

```diff
@@ -0,0 +1,45 @@
+# cddl-gen v. 0.3.0
+
+## Improvements:
+
+### CDDL language
+ * Add more support for hexadecimal/octal/binary numbers in CDDL. E.g. in size ranges or value limits
+ * Add support for ranges with three dots in CDDL files
+ * Change CDDL quantifier syntax from x**y to x*y
+ * Add support for $$ and $
+ * Add support for the .bits operator.
+
+### Command line
+ * Change command line name from cddl_gen to cddl-gen
+ * Add --dq as a shorthand for --default-max-qty
+ * Allow passing multiple cddl files via command line
+ * Allow adding the current git sha to generated file headers.
+ * Various changes to the logic for naming in code generation
+   * This can give different names of e.g. struct members compared with previous versions of cddl-gen
+ * Generate a cmake file when generating code.
+   * The target_cddl_source() Cmake function has been removed.
+   * See README.md for more info.
+
+### Python module
+ * Improve tags support in DataTranslator
+ * Improve decoding of cbor bstrs in DataTranslator
+   * Provide the raw string along with the decoded object.
+
+### C Libraries
+ * Refactor cbor_common.c
+ * Add support for indeterminate length arrays in cbor_decode.c.
+
+### Tests
+ * Move all cddl files into tests/cases
+ * Run tests with twister instead of custom bash scripts
+ * Add tests for draft-ietf-suit-manifest-14
+
+## Bugfixes:
+ * cbor_encode.h: Add missing '.' in tstrx_put and tstrx_put_term
+ * cbor_encode.c: Fix get_result_len and add big-endian support
+ * cddl_gen.py: Fix bug where range checks were dropped
+ * cbor_decode.c: Fix bug where a 'nil' could be interpreted as true
+ * cddl_gen.py: Fix parsing of ranges with no max (n* or n**)
```

Comment:
Similar to the above about inline for asterisks to avoid markdown syntax converting them unintentionally.